pkg/ruler/api_test.go

```diff
@@ -43,22 +44,22 @@ func cleanup(t *testing.T) {
 }
 
 // request makes a request to the configs API.
-func request(t *testing.T, method, urlStr string, body io.Reader) *httptest.ResponseRecorder {
+func request(t *testing.T, handler http.Handler, method, urlStr string, body io.Reader) *httptest.ResponseRecorder {
```

Comment:
Please include this change in the commit description, or in a separate commit if that makes sense.

Reply:
I've split this into a different commit.